src/widgets/chart.rs

```diff
@@ -154,6 +157,7 @@ struct ChartLayout {
 }
 
 /// A widget to plot one or more dataset in a cartesian coordinate system
+/// It will draw a legend box listing only datasets with a name set
```

Comment:
```suggestion
/// A widget to plot one or more dataset in a Cartesian coordinate system
///
/// It will draw a legend box listing every datasets that has a name set
```